internal/config/rtfiles.go

```diff
@@ -99,10 +99,6 @@ func (af assetFile) Data() ([]byte, error) {
 	return rt.Asset(string(af))
 }
 
-func (nf namedFile) Name() string {
```

Comment:
The `namedFile` type itself is not used either, so it should be also removed?

Reply:
removed